collectors/pod.go

```diff
@@ -39,7 +40,7 @@ var (
 	descPodInfo = prometheus.NewDesc(
 		"kube_pod_info",
 		"Information about pod.",
-		[]string{"namespace", "pod", "host_ip", "pod_ip", "node", "created_by"}, nil,
+		[]string{"namespace", "pod", "host_ip", "pod_ip", "node", "created_by", "owner_kind", "owner_name", "controller"}, nil,
```

Comment:
As a user I would expect the controller value to be something like "StatefulSetController", but it is actually a boolean value. I'd suggest, to rename it from "controller" to "owner_is_controller".